templates/themes/2/home.amp.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+
+<!---
+Copyright 2017 The AMPStart Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+{{#home}}
+<html ⚡>
+{{> ../utils/amp-page-head.snip.html}}
+<body>
+  {{> ../components/navbar/navbar.snip.html}}
+
+  {{#fullpage-hero}}
+    <figure class="ampstart-image-fullpage-hero m0 relative mb4">
+      <amp-img width="404" height="720" layout="responsive" src="{{narrow-img}}" media="(max-width: 415px)"></amp-img>
+      <amp-img height="720" layout="fixed-height" src="{{wide-img}}" media="(min-width: 416px)"></amp-img>
+      <header class="absolute top-0 right-0 left-0 center">
+        {{{heading-html}}}
+      </header>
+      {{#cta}}
+        <div class="absolute right-0 bottom-0 left-0 center">
+          <a class="h5 px4 py3 m3 inline-block text-decoration-none border" href="{{url}}">{{text}}</a>
+        </div>
+      {{/cta}}
+    </figure>
+  {{/fullpage-hero}}
```

Comment:
I think you could still use the existing fullpage hero component for this, 
(you could either pass in a class to the top figure -- to center ) or wrap this with a div and add center to that.
And add a CTA section in the mustache to render the button (Use anchor inside a button to get the style for free.)